pkgs/tools/text/uwuify/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, ... }:
+
+let
+  version = "0.1.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "uwu";
+  inherit version;
```

Comment:
Hm, is there a reason you're not declaring the version directly?

Reply:
Its used in more than one place (both version and v${}). I declared it this way so I didn't have to make it recursive or write it twice. Is this bad practice?

Reply:
I don't think so. I'm just not used to seeing it. LGTM then.